examples/onnx_ptq/README.md

```diff
@@ -111,12 +111,11 @@ quantize(
 
 ### Evaluate the quantized ONNX model
 
-The following evaluation requires the `val` directory of the [ImageNet dataset](https://www.kaggle.com/c/imagenet-object-localization-challenge/data). Alternatively, you can prepare it from [this](https://huggingface.co/datasets/mrm8488/ImageNet1K-val) Hugging Face dataset. Once you have it, the quantized ONNX ViT model can be evaluated on the ImageNet dataset as follows:
+The evaluation script automatically downloads and uses the [ILSVRC/imagenet-1k](https://huggingface.co/datasets/ILSVRC/imagenet-1k) dataset from Hugging Face. As this is a gated repository, a valid `HF_TOKEN` will be required for accessing the dataset. The quantized ONNX ViT model can be evaluated on the ImageNet dataset as follows:
```

Comment:
Does this mean that the user has to do `export HF_TOKEN=$HF_TOKEN` before running this script or do they need to set this somewhere else?